client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -123,24 +143,38 @@ export default function SitesDashboard() {
 
 	useEffect( () => {
 		// We need a category in the URL if we have a selected site
-		if ( selectedSiteUrl && ! category ) {
+		if ( sitesViewState.selectedSite && ! category ) {
 			setCategory( A4A_SITES_DASHBOARD_DEFAULT_CATEGORY );
-		} else if ( category && selectedSiteUrl && selectedSiteFeature ) {
-			page.replace( '/sites/' + category + '/' + selectedSiteUrl + '/' + selectedSiteFeature );
-		} else if ( category && selectedSiteUrl ) {
-			page.replace( '/sites/' + category + '/' + selectedSiteUrl );
-		} else if ( category ) {
+		} else if (
+			category &&
+			sitesViewState.selectedSite &&
+			selectedSiteFeature &&
+			selectedSiteFeature !== A4A_SITES_DASHBOARD_DEFAULT_FEATURE // If the selected feature is the default one, we can leave the url a little cleaner.
+		) {
+			page.replace(
+				'/sites/' + category + '/' + sitesViewState.selectedSite.url + '/' + selectedSiteFeature
```

Comment:
```suggestion
			page.replace(
				`/sites/${ category }/${ sitesViewState.selectedSite.url }/${ selectedSiteFeature }`
			);
```

What about using template literals like this?

Reply:
Great suggestion! https://github.com/Automattic/wp-calypso/pull/88644/commits/6460b8e6b220e8236a005754738e31e4c2b04f93